src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The main class for the bot
+ *
+ * @author Houten Teo
+ * @version CS2103T week 3
+ */
+public class Duke {
+    private static String format = "yyyy-MM-dd"; //default date
+    private MyList list;
+    private Storage storage;
+
+    public static void main(String[] args) {
+        new Duke().start();
+    }
+
+    /**
+     * Method to get the current format setting.
+     * @return the DateTimeFormatter
+     */
+    public static String getFormat() {
```

Comment:
great to see that you format your response to enhance readability of your information for your users!